src/mlpack/tests/callback_test.cpp

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Thanks, I really appreciate small fixes like this.  For this one though we should make sure to add a comment that, among other things, specifies the license, like this:

```
/**
 * @file callback_test.cpp
 *
 * Tests for ensmallen callback functionality.
 *
 * mlpack is free software; you may redistribute it and/or modify it under the
 * terms of the 3-clause BSD license.  You should have received a copy of the
 * 3-clause BSD license along with mlpack.  If not, see
 * http://www.opensource.org/licenses/BSD-3-Clause for more information.
 */
```

If you want, I can handle it during merge. :+1:

Reply:
Handled with 90014a6f9.